### ISSUE TITLE
Could you share where you get these pre-trained weights?

### ISSUE THREAD

**Issue by Zhong-eng** [ChuHan89/WSSS-Tissue#7]
Hi,

I think you provided an amazing method for cell image WSSS, but I'm not sure how you got the pre-trained weights for ResNet-38 and DeepLab v3. Could you tell me on which datasets they are pre-trained and where you found them? 

Thank you so much for your help! I'm really impressed with your work!

**Comment by ChuHan89** [comment]
Hi,

The pre-trained weights can be downloaded by this onedrive link.
https://1drv.ms/u/s!AgOtqK2ZncKlgoRobleElpBC5rbf7A?e=bDfqks

Chu

**Comment by Zhong-eng** [comment]
> Hi,
> 
> The pre-trained weights can be downloaded by this onedrive link. https://1drv.ms/u/s!AgOtqK2ZncKlgoRobleElpBC5rbf7A?e=bDfqks
> 
> Chu

Yes, I have downloaded these weights through the link on the README. Could you give me more information about how you got these weights? For example, on which dataset these weights are pre-trained, or some links to the websites you got them. Thanks for your help!
